Hw14

baldezhnik opened the pull request:
тут и 14 и 15 дз

baldezhnik committed:
add 1-14

baldezhnik committed:
Теперь выводится в приложение а не в логи 14-1

baldezhnik committed:
Добавленно задание 14-2

baldezhnik committed:
Добавленно задание 14-3

baldezhnik committed:
Добавленно задание 14-4

baldezhnik committed:
Переработано приложение.
Добавлен главный экран с выбором домашнего задания.
14 домашка перенесена в другую Activity
Перенесена как есть, в дальнейшем может подправлю для красоты.

baldezhnik committed:
Главное меню: добавлена кнопка перехода на активити 15 дз
Добавлено активити 15 дз

baldezhnik committed:
Добавлено задание 15-1

baldezhnik committed:
Добавлено задание 15-2

baldezhnik committed:
Добавлено задание 15-3

baldezhnik committed:
Добавлено задание 15-4

baldezhnik committed:
А на часах 2.35 am и я добавляю задание 15-5

OKurcheuskaya commented:
разные домашки должны быть в разных ветках

OKurcheuskaya commented on app/src/main/java/com/dsd/homework/MainActivity.kt:
```diff
@@ -9,7 +9,11 @@ class MainActivity : AppCompatActivity() {
+        val hw14 = HW14()
+        val tv: TextView = findViewById(R.id.textView)
+        tv.text = hw14.doHomeWork1()
+
```
не оставляй пустые строки

OKurcheuskaya commented on app/src/main/java/com/dsd/homework/MainActivity.kt:
```diff
@@ -14,9 +14,59 @@ class MainActivity : AppCompatActivity() {
+        val i :Int = (0..2).random()
+        textView7.text = hw14.zagadkaList[i].getZagadka()
+        when ((1..6).random()){
+            1-> {button2.text = hw14.zagadkaList[i].getOtvet()
```
попробуй убрать дублирование кода, путем создания метода

OKurcheuskaya commented on app/src/main/java/com/dsd/homework/MainActivity.kt:
```diff
@@ -12,9 +12,14 @@ class MainActivity : AppCompatActivity() {
-        val activ2 = Intent(this, HW14Activity::class.java)
-        startActivity(activ2)
+    fun activityHW14(view: View) {
+        val activity = Intent(this, HW14Activity::class.java)
```
наименование переменных должны быть говорящими. не activity a  intent

baldezhnik committed:
1. изменены названия переменных согласно замечанию
2. убраны лишние строки, оставлены те, которые помогают мне в навигации
3. переработана 14 домашка
 3.1 в HW14 отсался только класс загалки и список загадок все остальные функции перекочевали в HW14Activity
 3.2 3. переработана функция zapolnitZagadku согласно замечанию

baldezhnik commented:
> разные домашки должны быть в разных ветках

из-за того что у меня создан класс User и ListUser которые используются в 2-ух домашках, то мне бы пришлось в двух ветках его создавать, или ждать пока вы бы одобрили 14 домашку, потом соединять, крче каша.
последующие домашки постараюсь в разных ветках)
по остальным замечаниям внес изменения
